lib/map_elevation.dart

```diff
@@ -269,12 +267,10 @@ class ElevationGradientColors {
 }
 
 /// Geographic point with elevation
-class ElevationPoint {
-  /// Latitude and Longitude
-  lg.LatLng latLng;
-
+class InfoPoint extends lg.LatLng {
   /// Altitude (in meters)
   double altitude;
 
-  ElevationPoint(this.latLng, this.altitude);
+  InfoPoint(double latitude, double longitude, this.altitude)
+      : super(latitude, longitude);
 }
```

Comment:
Add a getter `latLng` which return the current instance for backward compatibility
Add a doc comment that deprecates it